### DISCUSSION TITLE
[Feature] add new lal with k8s configmap

### DISCUSSION THREAD

**Discussion by crazychengmm** [apache/skywalking#11548] [Q&A] [answered]
### Search before asking

- [X] I had searched in the [issues](https://github.com/apache/skywalking/issues?q=is%3Aissue) and found no similar feature requirement.

### Description

If skywalking start with docker in k8s, is it prossible to add new lol(xyz.yaml) with k8s configmap?
Or i need add xyz.yaml to resource, rebuild the mirror？

![image](https://github.com/apache/skywalking/assets/23517542/175d0135-c73a-493b-a3c2-3d20afa4162b)

### Are you willing to submit a pull request to implement this on your own?

- [X] Yes I am willing to submit a pull request on my own!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

**Comment by wu-sheng** [comment, marked as answer]
https://skywalking.apache.org/docs/main/v9.6.0/en/setup/backend/backend-docker/#extend-image

Use the ext dir, you could override any config files.

**Reply by crazychengmm** [reply]
Thanks, it work for me.
